### DISCUSSION TITLE
LOFTEE prediction for truncating variant & UTRannotator

### DISCUSSION THREAD

**Discussion by JMdeSteAgathe** [vidboda/MobiDetails#54] [Ideas]
Dear David and MD team,

I would find it quite usefull to have some cool VEP plugins in MD
- LOFTEE hg19 / hg38 predictions (they might differ from one another, sometimes)
- UTRannotator
Both can be retrieved through the ensembl rest API

LOFTEE:
> https://rest.ensembl.org/vep/human/hgvs/chrX(hg38):g.41341511G>A?/1&LoF=1/1&merged=1/1&hgvs=1
> &
> https://grch37.rest.ensembl.org/vep/human/hgvs/chrX(hg19):g.41200764G>A?/1&LoF=1/1&merged=1/1&hgvs=1

UTRannotator
> https://rest.ensembl.org/vep/human/hgvs/5:g.36877039_36877040delinsA?/1&UTRAnnotator=1/1&merged=1/1&hgvs=1

NB: the `/1&merged=1/1&hgvs=1` will allow you to match your refseq nomenclature to the corresponding result

**Comment by vidboda** [comment]
Hi @JMdeSteAgathe , looks interesting.

I need to check what I can do with this. On my todo list.

best,
